bufferserver/src/main/java/com/datatorrent/bufferserver/internal/DataList.java

```diff
@@ -710,35 +727,70 @@ private Runnable getStorer(final byte[] data, final int readingOffset, final int
         @Override
         public void run()
         {
-          int i = storage.store(identifier, data, readingOffset, writingOffset);
-          if (i == 0) {
+          if (uniqueIdentifier == 0) {
```

Comment:
What's the significance of zero? Can this be extracted into a constant that clarifies its importance rather than it being a magic number?